src/System.ServiceProcess.ServiceController/tests/SafeServiceControllerTests.cs

```diff
@@ -167,6 +167,7 @@ public static void SetServiceName_GetDisplayName()
         }
 
         [Fact]
+        [OuterLoop]
```

Comment:
could you please put a comment why we have this test as outerloop?

Reply:
See my https://github.com/dotnet/corefx/pull/39349/files#r301840543 on this: